app/views/layouts/application.html.erb

```diff
@@ -2,6 +2,16 @@
 <html lang="en">
 
   <head>
+  <% if Rails.env.production? %>
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-159214613-2"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+      gtag('config', 'UA-159214613-2');
```

Comment:
Can the token be replaced with a config value, probably from outside the app - maybe an env var? Would be worth talking through with @sfawcett123 how best to handle this?

The `<% if Rails.env.production?` could then be replaced with something like `<% if ENV['GA_ID'].present?` and could be test in development by setting the appropriate env var?